### PR TITLE
Publish only price, close #1480

### LINKED ISSUE #1480: Add support for publish only price
We have to add the new `scope` property to our  product publish update action `io.sphere.sdk.products.commands.updateactions.Publish`. The `scope` property is an enum with the literals `All` and `Prices`.

We have to add the new `scope` property to our `io.sphere.sdk.products.messages.ProductPublishedMessage' message too.

### PR DESCRIPTION
**Pull request by ccsalazarr-commercetools**

### CODE REVIEW

**Comment by ccsalazarr-commercetools** [comment]
@katmatt I have a question, in `ProductPublishedMessage`, the scope property should be `private final PublishScope scope;` or `private final String scope;`?

And I am not able to create an Integration test, I am not able to run on my local branch the `ProductPublishedMessageIntegrationTest`, but it passes in Travis.

**Comment by ccsalazarr-commercetools** [comment]
@katmatt could you review it please

File: commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package io.sphere.sdk.products;
+
+/**
+ * PublishScope.
+ */
+public enum PublishScope {
+
+    /* All is the scope by default to allow to publish all changes */
+    All,
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java]: Our convention is to use uppercase literals for enum, which then will be converted to the platform convention. So please change all enum literals to uppercase here.

File: commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
+package io.sphere.sdk.products;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import static org.apache.commons.lang3.StringUtils.capitalize;
+
+/**
+ * PublishScope.
+ */
+public enum PublishScope {
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java]: Sorry that I forgot it in my last comments: Each enum in our SDK must implement the `SphereEnumeration` interface.

File: commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+    /* ALL is the scope by default to allow to publish all changes */
+    ALL,
+
+    /* PRICES allow to publish only the prices */
+    PRICES;
+
+    @JsonValue
+    public String toScopeString() {
```
Comment (katmatt) [inline commercetools-models/src/main/java/io/sphere/sdk/products/PublishScope.java]: With my comment from above, you should be able to remove this method.

**Pull request merged**